beta/examples/tensorflow/common/object_detection/utils/input_utils.py

```diff
@@ -10,6 +10,8 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
+# pylint: disable=E1120
+# pylint: disable=E1124
```

Comment:
@andrey-churkin @alexsu52 is it more clear to disable pylint errors exactly in places where they appears? And using names like `# pylint: disable=no-value-for-parameter`

Reply:
I agree with the comment regarding the location of the `pylint: disable=...`. I put it on the top of the file because there are multiple places with the same type of `pylint: disable=...`. Although, it may be confusing.

Reply:
In exceptional cases, it is justified.